src/test/index.ts

```diff
@@ -1,22 +1,33 @@
 import startServer from "../connect";
 import { expect } from "chai";
+import { getRepository } from "typeorm";
+import { User } from "../entity/User";
 import * as dotenv from "dotenv";
 import * as request from "supertest";
 
+const createUserMutation = `mutation {
+  createUser(data: {name: "Name Test", email: "test@mail.com", password: "123456teste", birthDate: "06-05-1999"}){
```

Comment:
Não precisa alterar nesse PR, mas vc vai ver que cai ficar engessado esse texto sem usar variáveis do GraphQL. Quando vc for criar um teste e2e para o caso de "e-mail já existente", por exemplo, você não vai conseguir variar o e-mail aqui.